api-gateway/src/main/java/com/petclinic/bffapigateway/domainclientlayer/CustomersServiceClient.java

```diff
@@ -119,4 +113,47 @@ public Mono<OwnerDetails> deleteOwner (final long ownerId) {
                 .bodyToMono(OwnerDetails.class);
     }
 
+    public Mono<String> setPhotoOwner(PhotoDetails file, int id){
```

Comment:
We should have a discussion about refactoring this file into separate files by concern